### PR TITLE
Removes swarmers from the game

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
What the title says. But why?
I generally have a rule when making a contribution, that is "don't make the game less fun"
I'm not salting, I didn't die to a swarmer.
... Yet that's the problem. Swarmers are the griefiest antag in the game, but when you complain that they're annoying or unfun, you're doomed to hear "lol they can't even hurt you though."

WELL THAT ACTUALLY MAKES THEM WORSE. I would rather die to a hundred xenos and space dragons than be forced to untie myself in maintenance for 45 seconds while the shuttle leaves.

## Why It's Good For The Game
Unfun game modes should be removed from the game.
- Being griefed by swarmers is annoying
- Playing as a swarmer is not very exciting either. Click on iron.

lastly, because oranges authorized it

## Changelog

:cl:
del: Removes swarmers! The griefiest, lowest fun value antagonist is removed from the game.
/:cl:

### CODE REVIEW

**Review by MrMelbert — CHANGES_REQUESTED** [review]
as discussed in discord, remove them entirely

this pr doesn't do anything currently since the ruleset is disabled anyways

**Comment by JohnFulpWillard** [comment]
There's a lavaland NPC Swarmer which includes the beacon being included as megafauna (for achievement purposes). I don't know what is wanted there, but I wanted to bring it to light as soon as possible.

**Comment by Mothblocks** [comment]
> this pr doesn't do anything currently since the ruleset is disabled anyways

not true

**Comment by Mothblocks** [comment]
families and BB are the only things disabled on the [LRP] servers that arent disabled in code

File: strings/valentines.json
Comment (jlsnow301) [inline strings/valentines.json:74]: thanks i hate it

File: code/__DEFINES/achievements.dm
Comment (JohnFulpWillard) [inline code/__DEFINES/achievements.dm:80]: I think these defines should stay, just like king goat ones.

Comment (jlsnow301) [reply code/__DEFINES/achievements.dm]: done

**Comment by IndieanaJones** [comment]
Yeah this is fine really, I uh kinda screwed the pooch on Fikou's plan to make them better by having complete and total apathy towards this game for a couple of months/a year and I don't know if what was planned was going to even work out or just make them more like harmless annoying spiders, really.  Sometimes you just have to let things go.

**Review by MrMelbert — APPROVED** [review]

**Comment by Shadowflame909** [comment]
> EDIT: My suggestion would be having a traitor objective give you one to assist in the sabotage of some station structure? Not sure what structures would fit the bill but it could be neat

They can be turned into a Progression traitors final objective. So their less annoying and more "its time to leave its the apocalypse" like romerol zombies, and syndicate battlecruiser

**Comment by SethLafuente** [comment]
This is sad. Anyways

**Comment by realthomas69** [comment]
Ok this is epic

**Comment by necromanceranne** [comment]
Good, what sane person actually likes swarmers?

**Comment by SethLafuente** [comment]
> Good, what sane person actually likes swarmers?

I do ):

**Comment by wesoda25** [comment]
I think they're fascinating, I really like the language can we keep it?

**Comment by Shadowflame909** [comment]
keep the weak mob variants in the game

remove their replication ability. Thank you

**Comment by jlsnow301** [comment]
> keep the weak mob variants in the game
> 
> 
> 
> remove their replication ability. Thank you

image

**Comment by nianjiilical** [comment]
> > EDIT: My suggestion would be having a traitor objective give you one to assist in the sabotage of some station structure? Not sure what structures would fit the bill but it could be neat
> 
> They can be turned into a Progression traitors final objective. So their less annoying and more "its time to leave its the apocalypse" like romerol zombies, and syndicate battlecruiser

this would actually be pretty cool but i doubt anyone likes swarmers enough to code it (i like swarmers but i cant code)

**Comment by MrMelbert** [comment]
> > this pr doesn't do anything currently since the ruleset is disabled anyways
> 
> not true

image

I took this picture from live servers, is it not accurate?

**Comment by optimumtact** [comment]
people who downvoted this PR need their heads checked

File: code/__DEFINES/role_preferences.dm
Comment (MrMelbert) [inline code/__DEFINES/role_preferences.dm:137]: i'm pretty sure we leave removed antags in this list for a reason

File: code/modules/admin/sql_ban_system.dm
Comment (MrMelbert) [inline code/modules/admin/sql_ban_system.dm:327]: wrong one, from THIS list

Comment (optimumtact) [reply code/modules/admin/sql_ban_system.dm:327]: yes, you will break old bans if you don't

**Comment by Mothblocks** [comment]
> Oh I saw the 101 threat cost and thought it's permanently disabled

maybe im reading the wrong file then

File: code/modules/admin/sql_ban_system.dm
Comment (Mothblocks) [reply code/modules/admin/sql_ban_system.dm:327]: are you sure? i thought this was just the list for the html page, like how you can click "antagonist positions" and auto select everything

**Comment by jlsnow301** [comment]
> people who downvoted this PR need their heads checked

Swarmers must've eaten their silo

File: code/modules/admin/sql_ban_system.dm
Comment (optimumtact) [reply code/modules/admin/sql_ban_system.dm]: I am not sure

Comment (Mothblocks) [reply code/modules/admin/sql_ban_system.dm]: @MrMelbert @jlsnow301 I aaam pretty sure you can remove all of these, the existing bans are in SQL anyway, so if swarmers get added back (theoretically) the bans will persist

Comment (MrMelbert) [reply code/modules/admin/sql_ban_system.dm]: if you say so, then it's probably fine

**Comment by optimumtact** [comment]
```
_maps/map_files/IceBoxStation/IceBoxStation.dmm:15194:error: undefined type: /obj/effect/temp_visual/swarmer
_maps/map_files/IceBoxStation/IceBoxStation.dmm:15196:error: unknown type
_maps/map_files/IceBoxStation/IceBoxStation.dmm:25117:error: undefined type: /obj/effect/temp_visual/swarmer
_maps/map_files/IceBoxStation/IceBoxStation.dmm:25119:error: unknown type
```

**Comment by DrDiasyl** [comment]
Once got stamina locked by swarmers for 20 minutes, based PR.

**Comment by ANAME593** [comment]
swarmers are very weak in combat they are usually one tap or two tap simply kill them with a lit welder you probably wont die or just shoot them with your gun as for actual swarms of yellow swarmers just run away they cant hit you from ranged

File: code/modules/admin/sql_ban_system.dm
Comment (Jordie0608) [reply code/modules/admin/sql_ban_system.dm:327]: It's fine to remove from this list; existing role bans would only break if swarmers were added back with a different name for their define in `role_preference.dm`

**Comment by Shadowflame909** [comment]
Just had a cult round ruined by swarmers breaching, mass stunning, and cuffing cultists even though the crew was dead.


Maintainers were right. Swarmers are the most unfun obstruction ever designed in ss13. Like a sentient 5 minute brig sentence as an enemy.

**Comment by necromanceranne** [comment]
> Maintainers were right.  Like a sentient 5 minute brig sentence as an enemy.

Fucking nailed it. Put this on their tombstone.

**Comment by Mothblocks** [comment]
damnit i was exporting a video of some time i killed a guy as a swarmer before merging it

**Comment by Mothblocks** [comment]
https://file.house/W6J4.mp4

check out this no hit swarmer run

**Comment by Toastgoats** [comment]
The fact that you felt the need to say that you totally didn't get salty about dying to swarmers says everything that needs to be said about this pr.

**Comment by Mothblocks** [comment]
look at the video above u toastgoats i was gonna merge the pr and i was the swarmer

**Comment by Toastgoats** [comment]
> look at the video above u toastgoats i was gonna merge the pr and i was the swarmer

Oh I wasn't talking about you I was talking about the pr in general.

**Comment by SethLafuente** [comment]
May swarmers rest in piece together with hive mind and clock cult F

**Comment by jlsnow301** [comment]
> Maintainers were right. Swarmers are the most unfun obstruction ever designed in ss13.

Glad to see you've been converted

**Comment by optimumtact** [comment]
> The fact that you felt the need to say that you totally didn't get salty about dying to swarmers says everything that needs to be said about this pr.

image

**Comment by Toastgoats** [comment]
> > The fact that you felt the need to say that you totally didn't get salty about dying to swarmers says everything that needs to be said about this pr.
> 
> image

Orange man mean :'(

**Comment by SomeDongus** [comment]
agony

**Pull request merged**
